packages/docs/src/pages/pkg-webpack-loader.mdx

```diff
@@ -54,6 +54,12 @@ When using `@babel/preset-react` with the [automatic runtime](https://babeljs.io
 
 Defaults to `true`.
 
+### importSources: string[]
+
+Additional libraries that should be parsed as though they were `@compiled/react` imports. This is passed directly to `@compiled/babel-plugin`.
```

Comment:
nit: mix of present and past tense here. Could be reworded to "Additional libraries that should be parsed as though they are `@compiled/react` imports. "

Reply:
isn't this the [subjunctive mood](https://www.grammar-monster.com/glossary/subjunctive_mood.htm)?